package.json

```diff
@@ -51,8 +51,8 @@
   },
   "dependencies": {
     "@dcl-sdk/utils": "^1.2.8",
-    "@dcl/js-runtime": "7.4.10",
-    "@dcl/sdk": "7.4.15",
+    "@dcl/js-runtime": "^7.4.15",
```

Comment:
Bumping this to `^7.4.15`, as it is no longer certain that `@dcl/sdk@7.4.10` can be used here.